reviews/templates/reviews/committee_members_workload.html

```diff
@@ -0,0 +1,88 @@
+                <tr>
+                    <th>{% trans "Reviewer" %}</th>
+                    <th>{% trans "Traject" %}</th>
+                    <th>{% trans "Referentienummer" %}</th>
+                    <th>{% trans "Datum ingediend" %}</th>
+                    <th>{% trans "Gewenste einddatum" %}</th>
+                </tr>
+                </thead>
+                <tbody>
+                {% for decision in decisions %}
+                    <tr>
+                        <td>{{ decision.reviewer.first_name }} {{ decision.reviewer.last_name }}</td>
+                        <td>
+                            {% if decision.review.proposal.is_revision == True %}
+                                {% trans 'Revisie' %}
+                            {% elif decision.review.short_route == True %}
+                                {% trans 'Korte route' %}
+                            {% else %}
+                                {% trans 'Lange route' %}
+                            {% endif %}
+                        </td>
+                        <td> 
+                            <a style="color:blue;" href="{% url 'reviews:detail' pk=decision.review.pk %}">
+                                {{ decision.review.proposal.reference_number }} 
+                            </a> - {{decision.review.proposal.title}}
+                        </td>
+                        <td data-order="{{ decision.review.date_start|date:'c' }}">{{ decision.review.date_start|date:"j M Y" }}</td>
+                        {% if decision.review.date_should_end < today %}
+                            <td data-order="{{ decision.review.date_should_end|date:'c' }}"> 
+                                <font color='red'> 
+                                    {{ decision.review.date_should_end|date:"j M Y" }} 
+                                </font> 
+                            </td>
+                        {% else %}
+                            <td data-order="{{ decision.review.date_should_end|date:'c' }}">
+                                {{ decision.review.date_should_end|date:"j M Y" }}
+                            </td>
+                        {% endif %}
+                    </tr>
+                {% endfor %}
+                </tbody>
+            </table>
+            <br />
+            <h3 style="margin-bottom: 20px;">{% trans 'Werkverdeling overzicht van afgelopen jaar' %} </h3>
+            <table class="dt" data-language="{% datatables_lang %}">
+                <thead>
+                <tr>
+                    <th>{% trans "Reviewer" %}</th>
+                    <th>{% trans "Korte route" %}</th>
+                    <th>{% trans "Lange Route" %}</th>
+                    <th>{% trans "Revisie" %}</th>
+                </tr>
+                </thead>
+                <tbody>
+                    {% for key, value in counts_dict.items %}
+                        <tr>
+                            <th scope="row" style="text-align:center">{{ key }}</th>
```

Comment:
While the row header makes sense here, it's not on the other table. Maybe add it there as well?